.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,25 +1,47 @@
 ---
 name: Bug Report
 about: Something doesn't work correctly in Atmosphère.
-
+#assignees:
 ---
 
 ## Bug Report
 
+[ If any section does not apply, use nil. ]  
```

Comment:
What does "use nil" mean? This can be made more clear.

Reply:
Nil simply means that it's a not an applicable section. Will change to N/A.